src/App.tsx

```diff
@@ -1,26 +1,190 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { USER_ID, deleteTodo, editTodo, getTodos } from './api/todos';
+import { Header } from './components/Header';
+import { Main } from './components/Main';
+import { Footer } from './components/Footer';
+import { Todo } from './types/Todo';
+import { FilterParams, filterTodos } from './utils/filterTodos';
+import { ErrorNotification } from './components/ErrorsNotification';
+import { ErroMessage } from './utils/errorMessages';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMesage, setErrorMessage] = useState<ErroMessage>(
+    ErroMessage.NO_ERRORS,
+  );
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [selected, setSelected] = useState<FilterParams>(FilterParams.ALL);
+  const [isLoading, setIsLoading] = useState<number[]>([]);
+  const [shouldFocusInput, setShouldFocusInput] = useState(false);
+
+  const countActive = () => {
+    return todos.filter(todo => todo.completed === false);
+  };
```

Comment:
1. this is not count - this is filtering
2. If you need this filtering - please re-use your own code from `filterTodos`